app/models/space.rb

```diff
@@ -39,10 +40,24 @@ def metrics
     end
   end
 
+  def guesstimates_of_type(types)
+    return [] if graph.nil?
+
+    return graph['guesstimates'].select {|guesstimate| types.include? guesstimate["guesstimateType"]}
+  end
+
   def is_public?
     !self.is_private
   end
 
+  def is_searchable?
+    is_public? and
```

Comment:
use && instead of 'and'.  Ruby ``and`` is not meant for logic.
http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby